### PR TITLE
[EI-143] Add css framework whitelabel support

### PR DESCRIPTION
**Pull request by chartrandf**
## Summary of changes

- Improve `Plugins > examples` page design
- Add `Plugins > Templates` page (MUICSS + Materialize + Expose all-in-on template)
- Add CSS framework as example

### CODE REVIEW

File: documentation/static/examples/portal/css/materializecss.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        --secondary-main: ${secondary.main};
+        --secondary-light: ${secondary.light};
+        --secondary-dark: ${secondary.dark};
+        --secondary-contrastText: ${secondary.contrastText};
+      }
+    `;
+
+    document.getElementsByTagName('head')[0].appendChild(themeStyle);
+  }
+};
```
Comment (chartrandf) [inline documentation/static/examples/portal/css/materializecss.js:29]: Idea, should we expose a function from SDK to apply/add theme color to plugin content? This function could supporting Portal and WDA theme, still not sure 🤷‍♂️

File: documentation/docs/plugins/portal.md [hunk 2]
```diff
@@ -176,3 +174,160 @@ You can add custom code when the user is not using a custom tab. It can be usefu
 Please refer to the [SDK](./sdk) documentation to know how to inject custom code in the application.
 
 The background script is always running, even when the user is logged out. Please make sure to remove all related background tasks when the `onLogout` listener event is fired.
+
+## Icons Support
+
+To defined `icon` and `parentIcon` image, we support two types of values:
+1. Path to a black `.svg` icon (recommanded)
```
Comment (bogue) [inline documentation/docs/plugins/portal.md:181]:
```suggestion
1. Path to a black `.svg` icon (recommended)
```

File: documentation/docs/plugins/templates.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+![New PBX Dashboard Tab](/img/all-in-one-template-portal.jpg)
+```mdx-code-block
+</BrowserWindow>
+```
+
+<ButtonsTrySource source="https://github.com/wazo-communication/euc-plugins-boilerplate" />
+
+## Portal Templates
+
+The following templates are adapted for [E-UC Portal](https://github.com/wazo-communication/euc-plugins-boilerplate) and include whitelabel supports.
```
Comment (bogue) [inline documentation/docs/plugins/templates.md:34]:
```suggestion
The following templates are adapted for [E-UC Portal](https://github.com/wazo-communication/euc-plugins-boilerplate) and include whitelabel support.
```

File: documentation/docs/plugins/templates.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+### HTML/JS/Materialize
+
+This template uses vanilla javascript, HTML and [Materialize](https://materializeweb.com/). Materialize is a modern responsive CSS framework based on Material Design by Google.
```
Comment (bogue) [inline documentation/docs/plugins/templates.md:38]:
```suggestion
This template uses vanilla Javascript, HTML and [Materialize](https://materializeweb.com/). Materialize is a modern responsive CSS framework based on Material Design by Google.
```

File: documentation/docs/plugins/templates.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+<ButtonsTrySource
+    product="portal"
+    manifest="https://wazo-communication.github.io/euc-plugins-js-sdk/examples/portal/css/manifest.json"
+    source="https://github.com/wazo-communication/euc-plugins-js-sdk/tree/master/documentation/static/examples/portal/css/materializecss.html"
+/>
+
+### HTML/JS/MUICSS
+
+This template uses vanilla javascript, HTML and [MUI (css)](https://www.muicss.com/). MUI is a lightweight CSS framework that follows Google's Material Design guidelines. MUI is designed from the ground up to be fast, small and developer-friendly.
```
Comment (bogue) [inline documentation/docs/plugins/templates.md:56]:
```suggestion
This template uses vanilla Javascript, HTML and [MUI (css)](https://www.muicss.com/). MUI is a lightweight CSS framework that follows Google's Material Design guidelines. MUI is designed from the ground up to be fast, small and developer-friendly.
```

File: documentation/src/components/BrowserWindow/IframeWindow.tsx [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    <div style={{padding: 10}}>
+      <BrowserWindow
+        url={url}
+        style={{
+          minWidth: 'min(100%,45vw)',
+          width: 800,
+          maxWidth: '100%',
+          overflow: 'hidden',
+        }}
+        bodyStyle={{padding: 0}}>
```
Comment (bogue) [inline documentation/src/components/BrowserWindow/IframeWindow.tsx:24]: Definitely picking the nit here, feel free to disregard
```suggestion
        bodyStyle={{padding: 0}}
      >
```

File: documentation/static/examples/wda/iframe-bg-messaging/tab.html [hunk 7]
```diff
@@ -1,7 +1,14 @@
...
+    <style>
+        body {
+            padding: 40px;
+            font-family: sans-serif;
+            font-size: 20px;
+        }
+    </style>
 <body>
-    My custom tab
+    Send and receive message between backgroundScript and the current tab
```
Comment (bogue) [inline documentation/static/examples/wda/iframe-bg-messaging/tab.html:11]: For consistency's sake, used this form previously:
```suggestion
    Send and receive message between background script and the current tab
```

**Review by bogue — APPROVED** [review]

**Pull request merged**